nextjs/packages/next/build/webpack-config.ts

```diff
@@ -997,6 +1014,18 @@ export default async function getBaseWebpackConfig(
               ]
             : defaultLoaders.babel,
         },
+        ...(isServer
+          ? []
+          : [
+              {
+                issuer: /(mutations|queries)(?!.*\.client)/,
+                resource: /_resolvers/,
+                use: { loader: 'null-loader' },
+              },
+              // Because of packages/core/src/prisma-utils.ts
+              { test: /[\\/]npm-which[\\/]/, use: { loader: 'null-loader' } },
+              { test: /[\\/]cross-spawn[\\/]/, use: { loader: 'null-loader' } },
```

Comment:
```suggestion
                use: { loader: require.resolve('null-loader') },
              },
              // Because of packages/core/src/prisma-utils.ts
              { test: /[\\/]npm-which[\\/]/, use: { loader: require.resolve('null-loader') } },
              { test: /[\\/]cross-spawn[\\/]/, use: { loader: require.resolve('null-loader') } },
```

These should be resolved up front so the correct version is used / avoid relying on hoisting

Reply:
Thanks!